shared/haml/hamburger.haml

```diff
@@ -2,7 +2,7 @@
   options = { level: level, script_level: script_level, language: language, user_type: user_type, loc_prefix: loc_prefix, request: request }
   contents = Hamburger.get_hamburger_contents(options)
 
-#hamburger{class: contents[:visibility], tabindex: "0", 'aria-label': I18n.t('header_screen_reader_hamburger')}
+%button#hamburger{class: contents[:visibility], tabindex: "0", role: "navigation", 'aria-label': "Menu"}
```

Comment:
Intentional change from translated text to string "Menu"? And are we able to remove tabindex now that this is a button?